lascli/parser/workflows.py

```diff
@@ -10,18 +10,31 @@ def list_workflows(las_client: Client, max_results=None, next_token=None):
     return las_client.list_workflows(max_results=max_results, next_token=next_token)
 
 
-def create_workflow(las_client: Client, specification_path, error_config, **optional_args):
+def create_workflow(las_client: Client, specification_path, error_config, completed_config, **optional_args):
     specification = json.loads(pathlib.Path(specification_path).read_text())
-    error_config = json.loads(error_config) if error_config else None
-    return las_client.create_workflow(specification, error_config=error_config, **optional_args)
+    error_config = json.loads(pathlib.Path(error_config)) if error_config else None
+    completed_config = json.loads(pathlib.Path(completed_config)) if completed_config else None
+    return las_client.create_workflow(
+        specification,
+        error_config=error_config,
+        completed_config=completed_config,
+        **optional_args,
+    )
```

Comment:
Other parameters that are paths have the suffix 'path', should it not be here as well?